webpack/ForemanTasks/Components/TaskDetails/Components/TaskHelper.js

```diff
@@ -0,0 +1,54 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import React from 'react';
+import { FormattedRelative } from 'react-intl';
+
+const timeSince = date => {
```

Comment:
Isn't there a library which does this?

Reply:
I'm not sure its a good idea to import a whole library for something that can be solved in 20 lines

Reply:
> I'm not sure its a good idea to import a whole library for something that can be solved in 20 lines

I'm not sure I agree in this case:

1. This file contains 60 lines of code
2. If someone else is responsible for this code (and they do a good job) it means less work for us in term of future maintenance.

I would say if there is a library which is popular, lean and maintains properly I would choose a library.
If not, I would vote for having this code in core so we can reuse it in core and in other plugins.

Reply:
Uses 'humanize-duration' library now